juriscraper/opinions/united_states/territories/mp.py

```diff
@@ -0,0 +1,49 @@
+"""Scraper for Supreme Court of Northern Mariana Islands
+CourtID: mp
+Court Short Name: NMI
+Author: William Edward Palin
+History:
+  2023-01-21: Created by William Palin
+"""
+
+from datetime import date
+
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        year = str(date.today().year)[-2:]
+        self.url = f"https://www.cnmilaw.org/spm{year}.php#gsc.tab=0"
+        self.status = "Published"
+
+    def _process_html(self):
+        for s in self.html.xpath(".//tr/td/.."):
+            cells = s.xpath(".//td")
+            judges = cells[3].text_content()
+            judges = [
+                j.replace(" Jr.", "Jr.").strip(" *") for j in judges.split(",")
+            ]
+            judges = [j.replace("Jr.", " Jr.") for j in judges]
```

Comment:
What are we doing here? Can you add a comment or tweak that makes it obvious?

Reply:
comments added.  also - this has been renamed to NMI .